src/validation.cpp

```diff
@@ -5302,3 +5345,25 @@ Chainstate& ChainstateManager::ActivateExistingSnapshot(CTxMemPool* mempool, uin
     m_active_chainstate = m_snapshot_chainstate.get();
     return *m_snapshot_chainstate;
 }
+
+const CBlockIndex* ChainstateManager::GetSnapshotBaseBlock() const
+{
+    AssertLockHeld(::cs_main);
+    const auto blockhash_op{this->SnapshotBlockhash()};
+    if (!blockhash_op) return nullptr;
+    return m_blockman.LookupBlockIndex(*blockhash_op);
+}
+
+std::optional<int> ChainstateManager::GetSnapshotBaseHeight() const
+{
+    AssertLockHeld(::cs_main);
+    auto base{this->GetSnapshotBaseBlock()};
+    return base ? std::make_optional(base->nHeight) : std::nullopt;
+}
+
+bool ChainstateManager::IsAnyChainInIBD()
```

Comment:
03abde6150e9950e18edc00cb0541a33f714a231
```suggestion
bool ChainstateManager::IsAnyChainInIBD() const
```

Reply:
Fixed.